dev/com.ibm.ws.wsoc/src/com/ibm/ws/wsoc/SessionIdleTimeout.java

```diff
@@ -36,6 +36,7 @@ public class SessionIdleTimeout implements Runnable {
     public SessionIdleTimeout(String sessionID, long maxIdleTimeout, WsocConnLink wsocLink) {
         this.sessionID = sessionID;
         this.maxIdleTimeout = maxIdleTimeout;
+        Thread.dumpStack();
```

Comment:
I assuem this was here for some debugging and needs to get cleaned up?

Reply:
Yes, I definitely need to remove that.